### PR TITLE
Console rollup period fix

### PR DESCRIPTION
**Pull request by rikysya**
What: fix date ranges for current  and previous rollup periods on console UI

Why: according to https://storjlabs.atlassian.net/browse/V3-1517 and https://storjlabs.atlassian.net/browse/V3-1518

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: web/satellite/src/components/project/UsageReport.vue [hunk 1]
```diff
@@ -88,21 +88,19 @@ import { toUnixTimestamp } from '@/utils/time';
...
                 this.$data.dateRange.endDate = currentDate;
             },
             beforeRouteLeave: function(to, from, next) {
-                const currentDate = new Date();
-                const previousDate = new Date();
-                previousDate.setDate(1);
+				const currentDate = new Date();
 
-                this.$data.dateRange.startDate = previousDate;
+                this.$data.dateRange.startDate = new Date(currentDate.getFullYear(), currentDate.getMonth(), 1);
```
Comment (coyle) [inline web/satellite/src/components/project/UsageReport.vue]: out of curiosity, how does this work with timezones ?

File: web/satellite/src/components/project/UsageReport.vue [hunk 2]
```diff
@@ -144,7 +140,7 @@ import { toUnixTimestamp } from '@/utils/time';
                 onPreviousRollupClick: async function (event: any) {
                     const date = new Date();
                     const previousDate = new Date(date.getFullYear(), date.getMonth() - 1, 1);
-                    const currentDate = new Date(date.getFullYear(), date.getMonth(), 0);
+                    const currentDate = new Date(date.getFullYear(), date.getMonth(), 0, 23, 59, 59);
```
Comment (coyle) [inline web/satellite/src/components/project/UsageReport.vue]: midnight ?

File: web/satellite/src/components/project/UsageReport.vue (hunk 1 above)
Comment (rikysya) [reply web/satellite/src/components/project/UsageReport.vue]: good question. But I guess it's fine, cause these are not invoices it just shows usage relatively to user timezone

File: web/satellite/src/components/project/UsageReport.vue (hunk 2 above)
Comment (rikysya) [reply web/satellite/src/components/project/UsageReport.vue]: yes

**Pull request merged**